src/org/labkey/test/util/TestDataGenerator.java

```diff
@@ -322,10 +322,12 @@ public String writeTsvContents()
      * Creates a file containing the contents of the current rows, formatted in TSV.
      * The file is written to the test temp dir
      * @param fileName  the name of the file, e.g. 'testDataFileForMyTest.tsv'
-     * @return
+     * @return File object pointing at created TSV
      */
     public File writeData(String fileName)
     {
```

Comment:
I like this.
Would it make sense to push this down into testFileUtils, maybe do something similar with the projectFilesRoot location?